src/Microsoft.DotNet.Interactive/Events/ValueNamesProduced.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    public class ValueNamesProduced : KernelEvent
```

Comment:
This event name isn't quite appropriate anymore.

Reply:
renamed this already